### PR TITLE
accept API Key and Port Number via Enviroment Variable

### PR DESCRIPTION
**Pull request by mubarak23**
Change flow
- i install dotenv package
- create .env file with following variable
OPEN_AI_API_KEY=
PORT=
and update the index.js file to pull the variable from .env file

### CODE REVIEW

**Review by Cliftonz — APPROVED** [review]
Lgtm

File: server/index.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
 const { Configuration, OpenAIApi } = require("openai");
 const express = require("express");
 const cors = require("cors");
+
+require('dotenv').config
+
 const app = express();
-const PORT = 4000;
+const PORT = process.env.PORT;
```
Comment (Cliftonz) [inline server/index.js:8]: Can you add an .env.example file that has these variables in there?

**Review by Cliftonz — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 1 above)
Comment (mubarak23) [reply server/index.js:8]: Sure, i will added it right away

Comment (mubarak23) [reply server/index.js:8]: Added

**Review by Cliftonz — APPROVED** [review]

**Pull request merged**